test/WebJobs.Script.Tests/Description/FunctionInvokerBaseTests.cs

```diff
@@ -23,45 +22,59 @@ public class FunctionInvokerBaseTests
     {
         private MockInvoker _invoker;
         private TestMetricsLogger _metricsLogger;
-        private TestTraceWriter _traceWriter;
+        private TestLoggerProvider _testLoggerProvider;
 
         public FunctionInvokerBaseTests()
         {
             _metricsLogger = new TestMetricsLogger();
-            _traceWriter = new TestTraceWriter(TraceLevel.Verbose);
-            var config = new ScriptHostConfiguration();
-            config.HostConfig.AddService<IMetricsLogger>(_metricsLogger);
+            _testLoggerProvider = new TestLoggerProvider();
+
+            var scriptHostConfiguration = new ScriptHostConfiguration
+            {
+                HostConfig = new JobHostConfiguration(),
+                FileLoggingMode = FileLoggingMode.Always,
+                FileWatchingEnabled = true
+            };
+
+            ILoggerFactory loggerFactory = new TestLoggerFactory();
```

Comment:
nit: you can actually just use a 'normal' `LoggerFactory` here. You could then back out your changes to `TestLoggerFactory`

Reply:
Do we need TestLoggerFactory then?